zombienet_tests/functional/0002-parachains-disputes.zndsl

```diff
@@ -53,3 +53,6 @@ ferdie: reports polkadot_parachain_disputes_finality_lag is 0
 eve: reports polkadot_parachain_disputes_finality_lag is 0
 one: reports polkadot_parachain_disputes_finality_lag is 0
 two: reports polkadot_parachain_disputes_finality_lag is 0
+
+# Check participating in the losing side of a dispute logged
+alice: log line contains "Voted approval for a candidate that was concluded invalid." within 180 seconds
```

Comment:
```suggestion
alice: log line contains "Voted for a candidate that was concluded invalid." within 180 seconds
```

Reply:
In this test, the malus nodes vote against valid. Let's have both log messages checked , you'd need to modify 0003 where malus nodes vote for invalid.